api/src/controllers/users.js

```diff
@@ -50,17 +51,36 @@ async function getUsers(req, res) {
  * @sends: User added or error message
  */
 async function addUser(req, res) {
-  const newUser = new User({
-    givenName: req.body.givenName,
-    familyName: req.body.familyName,
-    email: req.body.email,
-    password: req.body.password,
-  });
+	User.findOne({ email: req.body.email }, async (err, user) => {
+		if (err) {
+			res.status(500).json({message: err});
+			return;
+		}
+
+		if (user) {
+			res.status(400).json({message: "Email already registered"});
+			return;
+		}
+
+		if (req.body.password != req.body.confirmPassword) {
+			res.status(400).json({message: "Passwords do not match"});
+			return;
+		}
 
-  newUser
-    .save()
-    .then(() => res.status(201).json("User added!"))
-    .catch((err) => res.status(400).json("Error: " + err));
+		const hashedPassword = await bcrypt.hash(req.body.password, 10);
+
+		const newUser = new User({
+			givenName: req.body.givenName,
```

Comment:
`givenName` and `familyName` fields have now been changed to `teamName` and `orgName` in the User model.

Reply:
This has been updated